backend/tests/v5/unit/models/roles.test.js

```diff
@@ -75,9 +83,230 @@ const testRevokeTeamspaceRoleFromUser = () => {
 	});
 };
 
+// --- JOBS RENAMED TO ROLES --- TO BE KEPT
+
+const testGetRolesToUsers = () => {
+	describe('Get Roles to users', () => {
+		test('should get list of roles within the teamspace with the users', async () => {
+			const expectedResult = [
+				{ _id: 'roleA', users: ['a', 'b', 'c'] },
+			];
+			const fn = jest.spyOn(db, 'find').mockImplementation(() => expectedResult);
+			const teamspace = 'ts';
+			const res = await Roles.getRolesToUsers(teamspace);
+			expect(res).toEqual(expectedResult);
+			expect(fn.mock.calls.length).toBe(1);
```

Comment:
why aren't we using `expect().toHaveBeenCalledTimes(1)`
and `expect().toHaveBeenCalledWith()` etc?

If it's because it's just copy and pasted, let's just change it whilst we're editing

Reply:
same with the functions below,